build-phar.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+
+if [ ! -f box.phar ]; then
+    wget https://github.com/humbug/box/releases/download/3.0.0-beta.4/box.phar -O box.phar
+fi
+
+# lock PHP to minimum allowed version
+composer config platform.php 7.2.0
+composer update --no-dev
+
+php box.phar compile
```

Comment:
Does this tool change the namespace inside the `phar` to avoid namespace conflict if the tool is installed in `vendor`?

Reply:
Not in the current state, but it's possible with `KevinGH\Box\Compactor\PhpScoper`.
Should I add it?

Reply:
It is (probably) not required, because our tool is designed to not load any code. If we load any code from the analyzed projects, I'd say that we made a mistake.

Reply:
OK good for me :+1: